app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -1346,9 +1346,10 @@ def get_record_cls(record)
       record.class.base_class.name
     else
       klass = case record
-              when Host, ExtManagementSystem then record.class.base_class
-              when VmOrTemplate then              record.class.base_model
-              else                            record.class
+              when ContainerNode, ContainerGroup, Container then record.class.base_class
+              when Host, ExtManagementSystem                then record.class.base_class
```

Comment:
This whole method seems to be mostly different paths to call `record.class.base_class.name` :confused:

Do we know which classes *don't* want to call `base_class`?